rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -861,8 +859,12 @@ error_code sys_usbd_get_descriptor(ppu_thread& ppu, u32 handle, u32 device_handl
 		return CELL_EINVAL;
 	}
 
-	u8* ptr = static_cast<u8*>(descriptor.get_ptr());
-	usbh.handled_devices[device_handle].second->device.write_data(ptr);
+	if (!descriptor)
```

Comment:
It's worth noting that an error may not occur if size is 0, but it may be investigated later.

Reply:
CELL_ENOMEM when the size is 0.